app/views/publishers/vacancies/job_applications.html.slim

```diff
@@ -0,0 +1,74 @@
+- content_for :page_title_prefix, current_organisation.name
+
+= render BannerComponent.new do
+  - if @vacancy.expires_on.past?
+    = govuk_breadcrumbs breadcrumbs: { "#{t("publishers.vacancies_component.expired.tab_heading")}": jobs_with_type_organisation_path(:expired), "#{@vacancy.job_title}": organisation_job_job_applications_path(@vacancy.id) }
+  - else
+    = govuk_breadcrumbs breadcrumbs: { "#{t("publishers.vacancies_component.published.tab_heading")}": organisation_path, "#{@vacancy.job_title}": organisation_job_job_applications_path(@vacancy.id) }
+  .govuk-grid-row
+    .govuk-grid-column-full
+      h2.govuk-caption-l = t(".heading")
+      h1.govuk-heading-l = @vacancy.job_title
+
+  .govuk-grid-row
+    .govuk-grid-column-one-quarter
+      .application-deadline class=("application-deadline--passed" if @vacancy.expires_on.past?)
+        h4.govuk-body.govuk-caption-s class="govuk-!-font-weight-bold govuk-!-margin-bottom-0"
+          = @vacancy.expires_on.past? ? t(".deadline.after") : t(".deadline.before")
+        span.govuk-body = expiry_date_and_time(@vacancy)
+
+    .govuk-grid-column-one-quarter
+      .application-count.govuk-tag class="govuk-tag--#{status_tag_colour("submitted")}"
+        .govuk-heading-l = applications.submitted.count
+        h3.govuk-caption-s = t(".review")
+
+    .govuk-grid-column-one-quarter
+      .application-count.govuk-tag class="govuk-tag--#{status_tag_colour("shortlisted")}"
+        .govuk-heading-l = applications.shortlisted.count
+        h3.govuk-caption-s = t(".shortlisted")
+
+    .govuk-grid-column-one-quarter
+      .application-count.govuk-tag class="govuk-tag--#{status_tag_colour("unsuccessful")}"
```

Comment:
Sorry my comment before didn't read the method properly. Passing in `:unsuccessful` is ok, we just don't need to to `:unsuccessful.sym` in the method, because it's already a symbol